module1/homework/lengthSort/sort.cpp

```diff
@@ -0,0 +1,31 @@
+#include "sort.hpp"
+
+#include <algorithm>
+
+bool compareLexicographically(const std::string& first_string,
+                              const std::string& second_string) {
+    std::size_t i = 0;
+    while ((i < first_string.size()) && (i < second_string.size())) {
+        if (tolower(first_string[i]) < tolower(second_string[i])) {
+            return true;
+        } else if (tolower(first_string[i]) > tolower(second_string[i])) {
+            return false;
+        }
+        i++;
+    }
+    return false;
+}
+
+bool compareSize(const std::string& first_string, const std::string& second_string) {
+    return first_string.size() < second_string.size();
+}
```

Comment:
it can be done as one predicate and in simpler way

Reply:
Thanks, done :)